### DISCUSSION TITLE
关于“语雀”上的资源使用优化讨论

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#47] [2️⃣ 聊项目]
目前“语雀”的整体使用下来，还是挺不错，有几个限制与优化的地方，可以考虑。

**1、利用“语雀”的运营与激励机制**

如下图，我们可以带着大家一起玩起来~

![image](https://user-images.githubusercontent.com/15010826/169959908-9104a6c4-8cee-4907-9523-108512945211.png)
![image](https://user-images.githubusercontent.com/15010826/169960170-b31297e7-81b8-4be3-ab2c-421cce68e8b7.png)
![image](https://user-images.githubusercontent.com/15010826/169960254-12548414-b5d5-47cb-9de6-604413f5b3b7.png)

**2、关于外链的分享功能**

免费用户，对外分享的链接是有时效性的，如果永久有效需要付费版。我们这个是公益付费版，还不包含这个功能。可以有两个做法：
- 找案例的相关团队，看能否争取一下 
- 购买付费版

![image](https://user-images.githubusercontent.com/15010826/169960640-f63b149a-aadf-4d23-a07f-13007beabaff.png)

![image](https://user-images.githubusercontent.com/15010826/169960486-9ad3b5c9-673e-41b3-a4ce-d8e4d27868d4.png)

大家看看，采取什么策略~

**Comment by bifenglin** [comment]
1. 利用“语雀”的运营与激励机制这个我看了，而且点赞是最简单最轻松来进行的。可以鼓动大家点赞起来，然后让大家阅读起来，当然必须有高质量文章，团队内部如何激励比较合适。我看过别人的激励方式是，每人交一两百元，然后创作一篇高质量文章后返还，也可以免费阅读其他人的高质量文章。当然这个只是为了促进大家创作。而且这部分完全可以当成内部读书分享会同步到外部。
2. 关于外链永久分享环节，得确定下是否是硬需求，若是硬需求可以开通。

**Comment by will-ww** [comment]
我现在的语雀上面，貌似可以永久对外公开分享了，和楼上的那幅图对比，不知道是产品升了级，还是有什么操作了~

![image](https://user-images.githubusercontent.com/15010826/170604595-6aa7b747-314f-4672-87f4-c784f84e6648.png)

**Reply by tyn1998** [reply]
哈哈要什么来什么

**Reply by xgdyp** [reply]
太神奇了，前几天发公众号的时候我还在纠结要不要加原文链接（加了过两天就失效了），现在就可以直接用了
